app/pods/components/agenda/agenda-detail/sidebar-item/component.js

```diff
@@ -29,8 +29,6 @@ export default class SidebarItem extends Component {
   @tracked isClickable = true;
   @tracked isRetracted = this.args.agendaitem.retracted;
 
-  @tracked renderDetails = null;
-
```

Comment:
Waarom is deze eruit ? dit was een fix dat de detailview ineens al deze data gaat ophalen voor alle agendapunten, ook al zijn die niet in view